addons/im_livechat/controllers/main.py

```diff
@@ -145,6 +145,11 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
                 mail_create_nosubscribe=False,
                 lang=request.env['chatbot.script']._get_chatbot_language()
             ).sudo().create(channel_vals)
+            member_operator = channel.channel_member_ids.filtered(lambda m: m.partner_id == channel.livechat_operator_id)
+            member_operator.sudo().create_history("bot" if member_operator.partner_id.chatbot_script_id else "operator")
+            # In case we have a partner has a visitor
```

Comment:
This comment is no longer true? (visitor member is the other member regardless of partner or guest)